packages/nimble-components/src/rich-text-editor/styles.ts

```diff
@@ -1,12 +1,163 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '@microsoft/fast-foundation';
-import { bodyFont, bodyFontColor } from '../theme-provider/design-tokens';
+import {
+    bodyFont,
+    bodyFontColor,
+    borderHoverColor,
+    borderRgbPartialColor,
+    borderWidth,
+    smallDelay,
+    standardPadding
+} from '../theme-provider/design-tokens';
 
 export const styles = css`
-    ${display('flex')}
+    ${display('inline-flex')}
 
     :host {
         font: ${bodyFont};
+        outline: none;
         color: ${bodyFontColor};
+        width: auto;
+        height: 100%;
+        flex-direction: column;
+        position: relative;
```

Comment:
We shouldn't be setting / requiring specific position modes for the component itself, i.e. for the `:host`.
The user should be able to specify how they want to position / layout the component without it breaking the contents.

for example if I set the `<nimble-rich-text-editor style="position:absolute"></nimble-rich-text-edittor>` it renders poorly taking up too much space and pushing the scrollbar offscreen:
![image](https://github.com/ni/nimble/assets/1588923/300120d1-41b5-45e9-ba13-c974d1b40377)

Reply:
I have removed the positioning style from the `:host` and updated the necessary styles to not to break the layout.